crates/fuel-core/src/graphql_api.rs

```diff
@@ -91,6 +92,7 @@ pub const DEFAULT_QUERY_COSTS: Costs = Costs {
     get_peers: 40001,
     estimate_predicates: 40001,
     dry_run: 12000,
+    storage_read_replay: 12000,
```

Comment:
```suggestion
    storage_read_replay: 40001,
```

I think we want to allow only one request like this